### PR TITLE
Temp disable 404 link tests as they often take upwards of 40s

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves N/A

The 404 link tests which run on live and tests the links in the navigation and the footer to ensure they do not 404. 

The way our tests are setup means that these tests run on every page type for every service (the NAV doesn't change per page type).

We are seeing these tests take upwards of 40s per page type, So for the frontpage canonical and amp these tests can take nearly 80s

An issue to introduce a new way to test for 404's on our pages https://github.com/bbc/simorgh/issues/5667

**Current Test Coverage:**

- The Psammead nav component has snapshots ensuring an a tag is rendered based on the passed in props
- The Simorgh Nav container has unit tests ensuring that a valid a tag is rendered with the passed in props.

### CODE REVIEW

File: cypress/integration/pages/testsForAllPages.js [hunk 1]
```diff
@@ -523,33 +523,6 @@ export const testsThatNeverRunDuringSmokeTestingForAllPageTypes = ({
...
-              failOnStatusCode: false,
-            }).then(resp => {
-              expect(resp.status).to.not.equal(404, `Received 404 for ${url}`);
-            });
-          });
-        });
-      });
-    }
-  });
+  // TODO: Move these 404 link tests to a different service, they take wayyyyyyyyy to long.
```
Comment (sareh) [inline cypress/integration/pages/testsForAllPages.js]:
```suggestion
```

Comment (sareh) [reply cypress/integration/pages/testsForAllPages.js]: Can you please open a Simorgh ticket to add checks for navigation and footer links for our services in a different format and link it to this PR?

**Comment by amywalkerdev** [comment]
Can you also double check we have a unit test that makes sure we have constructed the link URL correctly?

**Comment by sareh** [comment]
Can you please open a Simorgh ticket to add checks for navigation and footer links for our services in a different format and link it to this PR.

**Comment by chris-hinds** [comment]
> Can you also double check we have a unit test that makes sure we have constructed the link URL correctly?

We have sufficient unit test coverage in both simorgh and psammead.

**Comment by chris-hinds** [comment]
> Can you please open a Simorgh ticket to add checks for navigation and footer links for our services in a different format and link it to this PR.

Added this issue to the description

**Review by sareh — APPROVED** [review]

**Review by amywalkerdev — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**